src/openrct2/ride/ride.c

```diff
@@ -2032,7 +2032,7 @@ static void ride_update(sint32 rideIndex)
         ride->num_customers_timeout = 0;
 
         // Shift number of customers history, start of the array is the most recent one
-        memmove(ride->num_customers + 1, ride->num_customers, 9 * sizeof(*(ride->num_customers)));
+        memmove(ride->num_customers + 1, ride->num_customers, sizeof(ride->num_customers) - 1);
```

Comment:
This isn't right
`sizeof(ride->num_customers) = 10 * 2 = 20`
`sizeof(ride->num_customers) - 1 = 19`
You only want to move `9 * sizeof(ride->num_customers)`